src/routes/write-exercise/components/__tests__/InteractionSection.spec.tsx

```diff
@@ -72,4 +72,68 @@ describe('InteractionSection', () => {
     fireEvent.press(button)
     await waitFor(() => expect(getByTestId('popover').props.isVisible).toBeTruthy())
   })
+
```

Comment:
Can you add a test case for the bug you fixed, so if the correct solution is built up of multiple words?